docs/workflows/standalone/dorado_basecalling.md

```diff
@@ -0,0 +1,111 @@
+- **Model Type (fast):** (Fast Model) The fastest model, recommended when speed is prioritized over accuracy, such as for initial analyses or non-critical applications.
+
+### Example Manual Models:
+
+- `dna_r10.4.1_e8.2_400bps_sup@v4.2.0`
+- `dna_r10.4.1_e8.2_400bps_hac@v4.2.0`
+- `dna_r10.4.1_e8.2_400bps_fast@v4.2.0`
+
+??? Info "Supported Kit Names"
+
+    Ensure you use an accepted kit name in the `kit_name` parameter. The following kit names are supported in Dorado:
+
+    ```
+    EXP-NBD103, EXP-NBD104, EXP-NBD114, EXP-NBD114-24, EXP-NBD196, EXP-PBC001, EXP-PBC096, 
+    SQK-16S024, SQK-16S114-24, SQK-LWB001, SQK-MLK111-96-XL, SQK-MLK114-96-XL, SQK-NBD111-24, 
+    SQK-NBD111-96, SQK-NBD114-24, SQK-NBD114-96, SQK-PBK004, SQK-PCB109, SQK-PCB110, SQK-PCB111-24, 
+    SQK-PCB114-24, SQK-RAB201, SQK-RAB204, SQK-RBK001, SQK-RBK004, SQK-RBK110-96, SQK-RBK111-24, 
+    SQK-RBK111-96, SQK-RBK114-24, SQK-RBK114-96, SQK-RLB001, SQK-RPB004, SQK-RPB114-24, 
+    TWIST-16-UDI, TWIST-96A-UDI, VSK-PTC001, VSK-VMK001, VSK-VMK004, VSK-VPS001
+    ```
+
+    Select from these options to avoid input errors.
+
+### Inputs
+
+| **Terra Task Name** | **Variable** | **Type** | **Description** | **Default Value** | **Terra Status** |
+|---|---|---|---|---|---|
+| dorado_basecalling_workflow | **input_files** | Array[File] | Array of `POD5` files for basecalling | None | Required |
+| dorado_basecalling_workflow | **dorado_model** | String | Model accuracy or full model name (default: 'sup') | "sup" | Optional |
+| dorado_basecalling_workflow | **fastq_file_name** | String | Prefix for naming output FASTQ files | None | Required |
+| dorado_basecalling_workflow | **kit_name** | String | Sequencing kit name used (e.g., `SQK-RPB114-24`) | None | Required |
+| basecall_task.basecall | **cpu** | Int | Number of CPUs allocated | 8 | Optional |
+| basecall_task.basecall | **memory** | Int | Amount of memory to allocate (GB) | 32 | Optional |
+| basecall_task.basecall | **gpuCount** | Int | Number of GPUs to use | 1 | Optional |
+| basecall_task.basecall | **gpuType** | String | Type of GPU (e.g., `nvidia-tesla-t4`) | "nvidia-tesla-t4" | Optional |
+| dorado_basecalling_workflow | **fastq_upload_path** | String | Terra folder path for uploading FASTQ files | None | Required |
+| dorado_basecalling_workflow | **terra_project** | String | Terra project ID for FASTQ file upload | None | Required |
+| dorado_basecalling_workflow | **terra_workspace** | String | Terra workspace for final FASTQ file upload | None | Required |
+| dorado_basecalling_workflow | **paired_end** | Boolean | Indicates if data is paired-end | false | Optional |
+| dorado_basecalling_workflow | **assembly_data** | Boolean | Indicates if the data is for assembly | false | Optional |
+| dorado_basecalling_workflow | **file_ending** | String? | File extension pattern for identifying files (e.g., ".fastq.gz") | None | Optional |
```

Comment:
Would it make sense to re-order the rows in this table so that all required inputs are at the top, followed by optional inputs below (with them being in alphabetical order, as Terra displays them)?

Not sure if this is done for other workflow inputs tables but might be slightly helpful to the user